### PR TITLE
social network and IM fields 

### PR DESCRIPTION
**Pull request by xh3n1**

### CODE REVIEW

**Comment by skjnldsv** [comment]
Is it ready for review? :)

**Comment by xh3n1** [comment]
@skjnldsv If you have any suggestions, please let me know. 
PS: Speak now or forever hold your peace   :wink:

File: js/services/vCardProperties.js [hunk 1]
```diff
@@ -131,11 +133,17 @@ angular.module('contactsApp')
 			},
 			options: [
 				{id: 'FACEBOOK', name: 'Facebook'},
+				{id: 'GITHUB', name: 'GitHub'}
```
Comment (irgendwie) [inline js/services/vCardProperties.js]: missing **,** (comma)

File: js/services/vCardProperties.js [hunk 2]
```diff
@@ -97,9 +97,11 @@ angular.module('contactsApp')
 				meta:{type:['SKYPE']}
 			},
 			options: [
-				{id: 'IRC', name:  'IRC'},
-				{id: 'SKYPE', name:'Skype'},
-				{id: 'TELEGRAM', name:'Telegram'}
+				{id: 'IRC', name: 'IRC'},
+				{id: 'KIK', name: 'KiK'}
```
Comment (irgendwie) [inline js/services/vCardProperties.js]: same here for the **,** (comma)

**Comment by xh3n1** [comment]
@irgendwie done :wink:

**Pull request merged**